Neos.Eel/Resources/Private/Grammar/AbstractParser.peg.inc

```diff
@@ -12,7 +12,7 @@ namespace Neos\Eel;
  * source code.
  */
 
-require_once __DIR__ . '/../../../Resources/Private/PHP/php-peg/Parser.php';
+require_once __DIR__ . '/../Resources/Private/PHP/php-peg/Parser.php';
```

Comment:
Hm, must be a misconception on my side and I guess I have a clue on what it is (this just being the code template for the actual AbstractParser, which resides somewhere else and not being actually executed itself)... well, no objections from my side then :)